lib/services/catalog/commands/index.js

```diff
@@ -79,10 +79,14 @@ module.exports = function (service) {
 		if (!spec.source || !_.isString(spec.source)) {
 			throw new Error('spec.source String is required');
 		}
+		if (spec.sync && !_.isBoolean(spec.sync)) {
```

Comment:
Seems like line 89 would be sufficient.

This three line block looks like it might have the potential to make LOTS of things break.